app/validators/fee/concerns/fee_type_rules_creator.rb

```diff
@@ -23,13 +23,13 @@ def add_rule(*args)
         end
 
         def graduated_fee_type_only_rule
-          @graduated_fee_type_only_rule ||= \
+          @graduated_fee_type_only_rule ||=
             ['claim.case_type_id',
              :inclusion,
              CaseType.trial_fees.ids,
-             message: 'case_type_inclusion',
-             attribute_for_error: :fee_type,
-             allow_nil: true]
+             { message: 'case_type_inclusion',
```

Comment:
I'm sure it is OK but I'm just checking that this change was intended. Likewise below.

Reply:
Yes, I wasn't completely sure about this either but have checked and it seems to be OK.

The last three items of the array are passed as an `options` hash to a [Rule::Struct](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/lib/rule/struct.rb) via [this class](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/validators/fee/lgfs/fee_type_rules.rb). This change just makes it explicit that it's a hash.